.github/actions/nm-set-env/action.yml

```diff
@@ -39,6 +39,9 @@ runs:
       # testmo
       echo "XDG_CONFIG_HOME=/usr/local/apps" >> $GITHUB_ENV
       echo "PROJECT_ID=12" >> $GITHUB_ENV
+      # disable usage stats (writes to protected /usr/local/apps)
```

Comment:
This action is shared across all test jobs so it's better to run all the jobs against the change to see if anything is impacted.

Reply:
We should be good if remote push passes. This literally just changes whether we report usage data to a server